tests/cloner_test.go

```diff
@@ -2822,13 +2805,8 @@ var _ = Describe("all clone tests", func() {
 
 				By("Waiting for clone to be completed")
 				Expect(utils.WaitForDataVolumePhase(f, dataVolume.Namespace, cdiv1.Succeeded, dataVolume.Name)).To(Succeed())
-				By("Check host assisted clone is taking place")
 				pvc, err := f.K8sClient.CoreV1().PersistentVolumeClaims(dataVolume.Namespace).Get(context.TODO(), dataVolume.Name, metav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
-				suffix := "-host-assisted-source-pvc"
-				Expect(pvc.Annotations[controller.AnnCloneRequest]).To(HaveSuffix(suffix))
-				Expect(pvc.Spec.DataSource).To(BeNil())
-				Expect(pvc.Spec.DataSourceRef).To(BeNil())
```

Comment:
why was it removed?

Reply:
we no longer fall back to host assisted - https://github.com/kubevirt/containerized-data-importer/pull/2765
so now this test is just about the result being a success